features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -164,6 +165,10 @@
 
     private final IndexSettings indexSettings;
 
```

Comment:
If the property to enable it is called "forwarding", should the class be called "Forwarder", too?

Reply:
I debated this in my mind but somehow I didn't like Forwarder for the class name.  For ElasticFlowRepository it is forwarding to kafka but when it comes to KafkaFlowPersister it is just persisting.

Reply:
> Class names should be nouns or noun phrases. This means that the name of the class should be something that would be the subject of a verb. In the case of object-oriented design, methods would be the verbs that take place on the thing that the class is a representation of. Some words should be avoided.

Reply:
Modified to forwarder.